### ISSUE TITLE
traffic.megaphone.fm breaks many popular podcasts

### ISSUE THREAD

**Issue by sectary-want-holly** [lightswitch05/hosts#239]
For example: https://pca.st/podcast/5f07a9d0-b6e1-0134-106e-25324e2a541d

**Comment by lightswitch05** [comment]
Good thing `traffic.megaphone.fm` isn't in any of my lists then.

**Comment by lyrad87** [comment]
While the domain he provided is incorrect, `adserver.va3.megaphone.cloud` is being blocked by your list.

**Comment by lightswitch05** [comment]
Thanks for providing useful information @lyrad87 - I'll move `adserver.va3.megaphone.cloud` to the aggressive list in the next update

**Comment by lightswitch05** [comment]
https://github.com/lightswitch05/hosts/commit/afad0e31c08d3ec733091ea93b3e4bc0fdb09a49

**Comment by sectary-want-holly** [comment]
> While the domain he provided is incorrect, `adserver.va3.megaphone.cloud` is being blocked by your list.

That domain doesn’t break the podcast I highlighted above. As I stated previous, the domain blocking the podcast was $subject. 

Whilst the, frankly rude, maintainer of this list states this domain was not blocked the service I use - NextDNS - stated the domain was being blocked by that list. I can’t explain the discrepancy, nor do I care enough either. 

The issue was resolved by whitelisting the domain in $subject.

**Comment by lightswitch05** [comment]
@sectary-want-holly you can search the **entire history of my list** - and others - here: https://blocklist-tools.developerdan.com/entries/search?q=traffic.megaphone.fm

As you'll find, `traffic.megaphone.fm` has never been blocked in this project. However, it has a CNAME value for `adserver.va3.megaphone.cloud` - which was blocked in the primary list. I don't know anything about NextDNS - perhaps they block based on CNAME values? Using that same website I linked to above, you can see that as of today, it is no longer blocked in my primary list and was added to the aggressive list: https://blocklist-tools.developerdan.com/entries/search?q=adserver.va3.megaphone.cloud

Perhaps whitelisting `traffic.megaphone.fm` told NextDNS not to block that domain - regardless of the CNAME value (I know that is how the PiHole works at least).

------

On another topic, my issue template says this:

> I appreciate feedback on this project - both positive and negative. Regardless of the nature of the comment, keep it respectful. I really don't want to create a 'code of conduct', but if you need some guidelines on your expected behavior, checkout this 'Hey Friend' song first https://vimeo.com/338708688

Calling me - or others - rude is certainly not in the spirit of the project, please refrain from name calling if you wish to participate in this project.

**Comment by sectary-want-holly** [comment]
> Calling me - or others - rude

You are rude - and abrupt. It should be called out.

**Comment by sectary-want-holly** [comment]
> As you'll find, `traffic.megaphone.fm` has never been blocked in this project.

I _fully_ realise this. Hence my comment that NextDNS was identifying it from this source list, hence the issue being raised. It's clear that was a false positive from NextDNS.  Regardless, the issue is resolved.

**Comment by lightswitch05** [comment]
Locking this thread since it has devolved. If anyone has issues/questions with these domains in the future, please feel free to open a new issue.
